lib/features/chat/domain/helix_get_channel_emotes.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'helix_images.dart';
+
+part 'helix_get_channel_emotes.freezed.dart';
+part 'helix_get_channel_emotes.g.dart';
+
+@freezed
+class HelixGetChannelEmotesResponse with _$HelixGetChannelEmotesResponse {
```

Comment:
Honestly, we can convert this to a HelixGetChatEmotesResponse and make the channel specific fields nullable.